FruitFruit/FruitFruit/Controllers/BottomSheetViewController.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  BottomSheetViewController.swift
+//  FruitFruit
+//
+//  Created by 유재훈 on 2022/07/26.
+//
+
+import UIKit
+
+class BottomSheetViewController: UIViewController {
```

Comment:
```suggestion
final class BottomSheetViewController: UIViewController {
```

final을 설정하면 더이상 상속되지 않는다는 것을 명시할 수 있어요